stable/atlantis/templates/statefulset.yaml

```diff
@@ -223,6 +219,10 @@ spec:
       tolerations:
 {{ toYaml . | indent 8 }}
     {{- end }}
+      {{- if .Values.imagePullSecrets }}
+      imagePullSecrets:
+        - name: {{ .Values.imagePullSecrets }}
+      {{- end }}
```

Comment:
Since image pull secrets accepts multiple secrets, this should accept an array and range over it, outputting each secrets.

The indentation is slightly off, the `-` should be directly below the `imagePullSecrets` to match the rest of the chart. Above it's 2 spaces indented.

Reply:
Also will need to update the README with the new docs for `imagePullSecrets`